profiles/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+  ~  
+  ~ Licensed under the Apache License, Version 2.0 (the "License").
+  ~ You may not use this file except in compliance with the License.
+  ~ A copy of the License is located at
+  ~  
+  ~  http://aws.amazon.com/apache2.0
+  ~  
+  ~ or in the "license" file accompanying this file. This file is distributed
+  ~ on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+  ~ express or implied. See the License for the specific language governing
+  ~ permissions and limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>software.amazon.awssdk</groupId>
+        <artifactId>aws-sdk-java-pom</artifactId>
+        <version>2.0.0-preview-10-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>profiles</artifactId>
+    <name>AWS Java SDK :: Profiles</name>
+    <description>
+        The AWS SDK for Java - Profile module holds the AWS Profile classes
```

Comment:
Should we be less prescriptive with our definitions? Something like "Allows loading information from AWS configuration and credentials files: https://docs.aws.amazon.com/cli/latest/userguide/cli-config-files.html"

Reply:
fixed